model/countries/DE/DE-parsing-rules.yaml

```diff
@@ -33,6 +33,19 @@ capture_definitions:
           prefix: {regex_reference: kHouseNumberOptionalPrefixRe}
           parts: [ {regex_reference: kBuildingValueRe} ]
 
+  AddressLinesDecomposition:
+    capture:
+      output: street-address-alternative-1
+      parts:
+      - capture:
+          output: street-location
+          parts: [ {regex_fragment: '.*'} ]
+      - regex_fragment: '\n'
+      - capture:
+          output: address-overflow
+          parts: [ {regex_fragment: '[\s\S]+'} ]
```

Comment:
Could you explain the difference  between '.' and '\s\S'?

Reply:
`.` does not match new-line characters. `\s\S` matches everything.

Reply:
Done.